### ISSUE TITLE
Intermittent failed with [object Object] error on upload

### ISSUE THREAD

**Issue by AaronMT** [google-github-actions/upload-cloud-storage#231]
### TL;DR

Not sure why, but I ran into a `Error: google-github-actions/upload-cloud-storage failed with [object Object]` on an upload step. I have not seen this before and I've been using this action for several months. 

Run and workflow links below. Nothing complicated, I am uploading a single `JSON` file

[1] Run: https://github.com/mozilla-mobile/mobile-test-health/runs/5571311372?check_suite_focus=true
[2] Workflow: https://github.com/mozilla-mobile/mobile-test-health/blob/main/.github/workflows/daily.yml

### Expected behavior

Successful upload

### Observed behavior

`Error: google-github-actions/upload-cloud-storage failed with [object Object]`

### Action YAML

```yaml
      - uses: google-github-actions/upload-cloud-storage@v0
        if: hashFiles('output.json') != ''
        name: Upload Artifact (json) to Google Cloud Storage
        with: 
          path: ${{ steps.date.outputs.date }}_${{ matrix.configuration }}.json
          destination: mobile-reports/public/moz-mobile-test-health/${{ steps.date.outputs.day }}
```

**Comment by sethvargo** [comment]
Hi @AaronMT 

Thank you for opening an issue. Unfortunately the links you provided are both 404 (maybe private repos)?

That message should definitely be a string, not `[Object object]`. Can you share the [debug logs](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging).

**Comment by verbanicm** [comment]
@AaronMT I updated the error messaging and pushed a new version of the `upload-cloud-storage` action. Can you try again? v0 should use the latest.

**Comment by AaronMT** [comment]
I'll re-open/comment if I see it again.
